### PR TITLE
fix: wallet validation

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Fixed small bugs in validation
Fixed small bugs in wallet display of coinbases and one-sided payments

### CODE REVIEW

**Comment by hansieodendaal** [comment]
```
FAIL [   0.380s] minotari_wallet::wallet_integration_tests output_manager_service_tests::service::test_txo_validation
```

**Comment by SWvheerden** [comment]
> ```
> FAIL [   0.380s] minotari_wallet::wallet_integration_tests output_manager_service_tests::service::test_txo_validation
> ```

yes I can see that, busy looking into it

File: base_layer/wallet/src/base_node_service/config.rs [hunk 1]
```diff
@@ -40,7 +40,7 @@ pub struct BaseNodeServiceConfig {
 impl Default for BaseNodeServiceConfig {
     fn default() -> Self {
         Self {
-            base_node_monitor_max_refresh_interval: Duration::from_secs(90),
+            base_node_monitor_max_refresh_interval: Duration::from_secs(30),
```
Comment (hansieodendaal) [inline base_layer/wallet/src/base_node_service/config.rs:43]: We need to update the default value in the corresponding config.toml file as well

**Review by hansieodendaal — DISMISSED** [review]
utACK

File: common/config/presets/d_console_wallet.toml [hunk 2]
```diff
@@ -157,7 +157,7 @@ event_channel_size = 3500
 [wallet.base_node]
 # Configuration for the wallet's base node service
 # The refresh interval (default = 3 s)
```
Comment (hansieodendaal) [inline common/config/presets/d_console_wallet.toml]:
```suggestion
# The refresh interval (default = 30 s)
```

**Pull request merged**